### ISSUE TITLE
4'th world causing longer startup time when it has command blocks

### ISSUE THREAD

**Issue by spetznack** [Multiverse/Multiverse-Core#1319]
How to replicate: 
1: Startup a clean server using either bukkit or spigot (I tested both). 
2: You got 3 worlds Survival, nether and the_end. Using survival for this example...
3: Create 5000 command blocks anywhere in the world (I tested at 200 from spawn). 
4: I tried outside game using McEdit and also ingame using world edit.. 
5: Stop the server.
6: Try to start up the server now. Startuptime should be normal (1-5 seconds)
7: Stop the server. 
8: Install multiverse, try again. Startuptime should still be normal.. 
9: Stop the server. 
10: Make a copy the survival world folder in the same place and name it test. (or w\e)
11: Startup the server. 
12: Type /mv import test NORMAL
13: Stop the server.
14: Start the server and see that it uses much longer time. 

tl;dr version: 
Having 5000 command blocks in the first world= normal startup time
Having 5000 command blocks in the 4'th world= much longer startup time! 

My (rented) server is restarting automaticly because it uses too long time to startup, this is the cause. disabling the world's autoload works, but it is a very important world in my server. I need a fix for this!

What is the problem with the 4'th world and command blocks? 

Thank you.
